ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/DeploymentVulnerabilitiesTable.tsx

```diff
@@ -22,6 +27,30 @@ import PendingExceptionLabelLayout from '../components/PendingExceptionLabelLayo
 import PartialCVEDataAlert from '../../components/PartialCVEDataAlert';
 import { FormattedDeploymentVulnerability } from './table.utils';
 
+export const tableId = 'WorkloadCvesDeploymentVulnerabilitiesTable';
+export const defaultColumns = {
+    os: {
```

Comment:
What do you think about `operatingSystem` in case column heading changes from unique **OS**?
Find in File has 4 results for `dataLabel="Operating system"` in Vulnerabilities including 2 files above

Reply:
➕